timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -568,6 +568,7 @@ def _train_multi(
         if time_limit is not None and len(models) > 0:
             time_limit_model_split /= len(models)
 
+        # TODO: Allow HPO only for some models?
```

Comment:
@canerturkmen, do you think we should we allow users to train some models with HPO and other models without HPO? Currently this is not supported but seems like a rather straightforward fix.

Reply:
I agree it sounds like we should allow this (few people would do HPO for ETS, etc.)